clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/SubnetController.groovy

```diff
@@ -31,10 +32,27 @@ class SubnetController {
   List<SubnetProvider> subnetProviders
 
   @RequestMapping(method = RequestMethod.GET)
-  Set<Subnet> list() {
-    subnetProviders.collectMany {
+  Map<String, Set<Subnet>> list() {
```

Comment:
Just spotted your comment on the companion PR. Let me have another look at it, and will ping this thread again shortly.